client/src/pages/api/og.tsx

```diff
@@ -0,0 +1,163 @@
+import { ImageResponse } from '@vercel/og';
+import clsx from 'clsx';
+import { NextRequest } from 'next/server';
+
+export const config = {
+  runtime: 'experimental-edge', // Can likely switch to "edge" in Next 14
+};
+
+const interFontBoldImport = fetch(new URL('../../../assets/Inter-Bold.ttf', import.meta.url)).then(
+  (res) => res.arrayBuffer()
+);
+const interFontMediumImport = fetch(
+  new URL('../../../assets/Inter-Medium.ttf', import.meta.url)
+).then((res) => res.arrayBuffer());
+
+export default async function handler(req: NextRequest) {
+  try {
+    const [interFontMedium, interFontBold] = await Promise.all([
+      interFontMediumImport,
+      interFontBoldImport,
+    ]);
+    const { searchParams } = new URL(req.url);
+
+    const title = searchParams.get('title');
+    const description = searchParams.get('description');
+    const primaryColor = searchParams.get('primaryColor');
+    const lightColor = searchParams.get('lightColor');
+    const darkColor = searchParams.get('darkColor');
+    const isDark = searchParams.get('isDark');
+    const logoLight = searchParams.get('logoLight');
+    const logoDark = searchParams.get('logoDark');
+
+    const isDarkMode = Boolean(isDark);
```

Comment:
This will make any truthy value of `isDark`, such as `isDark=false`, result in `isDarkMode=true`. Maybe you should use this?
```suggestion
    const isDarkMode = isDark === "true";
```